### DISCUSSION TITLE
Add threading to top level pull request comments

### DISCUSSION THREAD

**Discussion by fitz-vivodyne** [github/roadmap#1067] [Announcements]
See https://github.com/github/roadmap/issues/552 which was closed as outdated.

Not being able to have threaded discussions on pull requests is very annoying.

There's something of a workaround in being able to comment/start a thread on unrelated code, but it would be a much better user experience if it just worked.

**Comment by ankneis** [comment]
Hi @fitz-vivodyne! Thank you for your participation - we will be closing this specific Discussion. Here are a number of feedback discussions that cover this topic—feel free to upvote and share your feedback where it fits best, or create a new discussion if needed!

- https://github.com/orgs/community/discussions/8469
- https://github.com/orgs/community/discussions/5633
